.goreleaser.yaml

```diff
@@ -0,0 +1,51 @@
+---
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    # Keep mod files clean
+    - go mod tidy -v
+
+builds:
+  - main: main.go
+    binary: bin/diff
+    env:
+      - CGO_ENABLED=0
+    flags: -trimpath
+    ldflags: -s -w -X {{ .ModulePath }}/cmd.Version={{ .Version }}
+    targets:
+      # helm supported targets
+      # https://github.com/helm/helm/blob/main/Makefile#L4C1-L4C1
+      - darwin_amd64
+      - darwin_arm64
+      - linux_386
+      - linux_amd64
+      - linux_arm
+      - linux_arm64
+      - linux_ppc64le
+      - linux_s390x
+      - windows_amd64
+      # additional targets
+      - freebsd_amd64
+
+archives:
+  - format: tgz
+    name_template: '{{ .ProjectName }}-{{ tolower .Os }}-{{ .Arch }}'
```

Comment:
It turned out we had to use `macos` instead of `darwin` in the archive name.

https://github.com/databus23/helm-diff/pull/517/files#diff-42e26dc67aed8aa3edb2472b4403288c1699fb6dc47419b9a475f0f224fe4689R7

Reply:
This is correct - I missed this.